packages/ui/src/working-groups/components/OpeningsList/OpeningsList.tsx

```diff
@@ -6,7 +6,7 @@ import { UpcomingWorkingGroupOpening, WorkingGroupOpening } from '@/working-grou
 import { OpeningsListRow } from './OpeningsListRow'
 
 export interface OpeningsListProps {
-  openings: WorkingGroupOpening[] | UpcomingWorkingGroupOpening[]
+  openings: Array<WorkingGroupOpening | UpcomingWorkingGroupOpening>
```

Comment:
This line changes the type

*   from: (Array of WGO) _or_ (Array of UWGO)
*   to: Array of (WGO _or_ UWGO)

The intention was to distinguish that as you can pass an array of single types and not an array of a mixed types.

ps.: TS is building and I have this:

![](https://user-images.githubusercontent.com/247363/121023310-74511680-c7a3-11eb-89f0-dea4662d47bb.png)

Reply:
ps.: The project builds, so maybe it is some problem with VSCode?

The [TS playground detects similar situation](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgILIN4FgBQz-LAAmAXMgM5hSgDmuAvrrqJLIigEKa4HIgD2YAJKk+AVwC2AI2gMmOBPxCVkAGWCUAwvwkAHZAF5kACmCQJ5MqgDaAXWQAfZBzsBKQwD5ueAlAhgxKBBCc3IAOgk4XVNzT29eXgB6ROQAFQBPXQgAcTE4KCIeBPxgGBMAcmJywmCzCAl3bB9i3j8AoORtPVQY+tci4sZmhLbA4K7dDl6GgeR6fpwh3EVlME6dXXQjaat3Ay8MUY66iTDiIZWVCa5tk7IOPYOj4IBlKlpjEPqwgWEid3oQA) the type correctly